go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/opencontainers/go-digest
 
-go 1.13
+go 1.17
```

Comment:
Do we want to update this version to 1.17? I discovered Yesterday that they somewhat "silently" started to enforce Go versions based on this in go1.16.5.
So far, it _seems_ limited to what version can be used to manage the module, but not sure if they will enforce it in other ways;

on go1.16.4:

```bash
go mod tidy
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/golang/protobuf v1.5.2
go: downloading github.com/opencontainers/go-digest v1.0.0
```

on go1.16.5:

```bash
go mod tidy
go mod tidy: go.mod file indicates go 1.17, but maximum supported version is 1.16
```

It's not mentioned in the release notes, but is indeed in the milestone: https://github.com/golang/go/issues/46144

Reply:
Ok, I'll leave this at 1.13. I don't think we have anything using newer features than that.